config/config.exs

```diff
@@ -1,5 +0,0 @@
-use Mix.Config
-
-if Mix.env() == :test do
-  config :elixir_auth_google, httpoison: ElixirAuthGoogle.HTTPoison.InMemory
```

Comment:
I've in-lined this in the lib/elixir_auth_google.ex `@httpoison` module attribute so we don't need the file. 👍 #35